js/components/urlBar.js

```diff
@@ -204,8 +197,13 @@ class UrlBar extends ImmutableComponent {
       case KeyCodes.BACKSPACE:
         this.hideAutoComplete()
         break
+      // Do not trigger rendering of suggestions if you are pressing alt or shift
+      case KeyCodes.ALT:
+      case KeyCodes.SHIFT:
```

Comment:
I have added two more. If you think I missed some, please let me know I will add :)